### DISCUSSION TITLE
Kaios Web Suite

### DISCUSSION THREAD

**Discussion by goldblatty** [community/community#127351] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A great idea. Managed to install the app on my phone and the extension on chrome. how do i get them to connect? The app asks to scan a qr code?

**Comment by ebndev** [comment]
Hi @goldblatty,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from Kaios Web Suite. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

P.S - if you don't mind me asking, how did you end up on GitHub Discussions? We have quite a few folks wanting to reach out to developers in specific projects so we'd love to know what led you here. Thanks!

**Comment by goldblatty** [comment]
Thank you for your helpful reply

I am not a technician but I am quite good at getting into to inside of
phones and PCs etc

For various reasons I bought a kaios phone (I am used to android) and want
to operate it from my pc (On android there is an app called myphoneexplorer
which is brilliant)

Searching around I bumped into this


Rabbi Y Goldblatt
+61 488   061 825 (Aus)
+44 7411 033 550 (UK)
